tests/unit_tests/admin/test_admin.py

```
@@ -0,0 +1,36 @@
+@pytest.fixture
+def admin_blueprint():
+    mock_db = Mock()
+    mock_login_methods = Mock()
+    mock_locale_func = Mock()
+    return create_admin_blueprint(mock_login_methods, mock_db, mock_locale_func)
+
+@patch('platzky.admin.admin.render_template')
+def admin_panel_renders_login_when_no_user(mock_render_template, admin_blueprint):
+    with admin_blueprint.test_request_context('/admin/'):
+        session['user'] = None
+        response = admin_blueprint.view_functions['admin_panel']()
```

Ruff (0.8.2)

Local variable `response` is assigned to but never used

Remove assignment to unused variable `response`

(F841)